WorkloadGenerator/User.py

```diff
@@ -0,0 +1,26 @@
+import aiohttp
+from UserCommands import UserCommands
+
+
+class User:
+
+    def __init__(self, ip, port, user_id, commands, loop):
+        self.user_id = user_id
+        self.client = aiohttp.ClientSession(loop=loop)
+        self.command_handler = UserCommands(ip, port, self.client)
+        self.commands = commands
+
+    async def processCommands(self):
+        resps = []
+        for command in self.commands:
+            print(f'{self.user_id} processing {command}')
+            # TODO Update so that this saves the amount of time taken to run each command and which command was run for analysis
+            resp = await self.command_handler.handleCommand(command)
+            resps.append(resp)

```

Comment:
It'd probably be helpful to output the response received as well as the above print just so we can see if things are coming back as expected. At least for the debugging phase.

Reply:
I actually have this being covered in the UserCommand class instance the User creates where it prints the response each time :)

Reply:
Don't know how I missed that. Yep.